src/components/Banner/Banner.css

```diff
@@ -0,0 +1,145 @@
+
+
+.app__banner-container{
+    background: var(--color-dark);
+    color: var(--color-light);
+    padding:0 3rem;
+    justify-content: space-between;
+    gap: 2rem;
+}
+
+.app__banner-left{
+    flex:1;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content:center;
+    align-self: stretch;
+    background-image: linear-gradient(0deg, rgba(22, 21, 21,0.9) 1%,rgba(21,21,21,.01) 99%);
+}
+
+
+.app__banner-subHeading{
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    margin:2rem 0;
+    font-size: .7rem;
+}
+.app__banner-heading{
+    font-family: var(--font-aguafina);
+    font-size: 4rem;
+    line-height: 1;
+    letter-spacing: .25rem;
+    font-weight: 600;
+    
+}
+
+.app__banner-heading_2{
+
+    font-size: 6rem;
+    line-height: 1;
+    font-weight: 700;
+    margin-bottom: 1rem;
+}
+
+.app__banner-cta{
+    border: 2px solid var(--color-light);
+    outline: none;
+    background: var(--color-light);
+    color: var(--color-dark);
+    border-radius: 2rem;
+    font-size: 1rem;
+    padding: 0.6rem 1rem;
+    box-shadow: none;
+    transition: all .5s ease-in;
+    margin-bottom: 2rem;
+}
+
+.app__banner-cta:hover{
+    border: 2px solid var(--color-light);
+    background: var(--color-dark);
+    color: var(--color-light);
+
+}
+
+.app__banner-right{
+    flex: 1;    
+    align-self: stretch;
+    position: relative;
+}
+
+.app__banner-right::after{
+    content: "";
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-image: linear-gradient(0deg, rgba(22, 21, 21,0.9) 1%,rgba(21,21,21,.01) 99%);
+}
+
+.app__banner-right img{
+    align-self: flex-end;
+}
+
+
+@media screen and (max-width:920px) {
+    .app__banner-heading{
+        font-size: 4rem;
+    }
+    .app__banner-heading_2{
+        font-size: 4rem;
+    }
+}
+
+@media screen and (max-width:700px) {
+    .app__banner-left{
+        text-align: center;
+    }
+    .app__banner-container{
+        flex-direction: column;
+    }
+
+    .app__banner-left{
+        align-items: center;
+        justify-content: center;
+    }
+}
+
+
+
+@media screen and (max-width:400px)
+{
+   
+    .app__banner-subHeading{
+        font-size: .5rem;
+    }
+    .app__banner-heading{
+        font-size: 3rem;
+    }
+    .app__banner-heading_2{
+        font-size: 3.5rem;
+    }
+    .app__banner-cta{
+        font-size: .7rem;
+    }
+}
+
+@media screen and (max-width:350px)
+{
+    .app__banner-container{
+        gap: .25rem;
+    }
+   
+    .app__banner-subHeading{
+        font-size: .5rem;
+    }
+    .app__banner-heading{
+        font-size: 2rem;
+    }
+    .app__banner-heading_2{
+        font-size: 2.25rem;
+    }
+    .app__banner-cta{
+        font-size: .75rem;
+        padding: .25rem .5rem;
+    }
```

Comment:
can directly change the root font size since default units are in rem they get resized automatically